doc/index.rst

```diff
@@ -63,6 +63,8 @@ an open source platform.
    </ul>
 
 
+.. include:: CONTRIBUTING
```

Comment:
We already have a contribute document (developer-guides/contribute_guidelines.html) so this include shouldn't be here on the home page.

Reply:
Yes, but I wanted to keep this file at the top-level for a couple of reasons:
1. It's common practice to find such file at the top-level folder
2. This is directly pointed at by the [Technical Charter](https://projectacrn.org/technical-charter/)